shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/UpstreamCheckService.java

```diff
@@ -456,30 +461,20 @@ public void fetchUpstreamData() {
     }
 
     /**
-     * listen {@link SelectorCreatedEvent} add data permission.
-     *
-     * @param event event
-     */
-    @EventListener(SelectorCreatedEvent.class)
-    public void onSelectorCreated(final SelectorCreatedEvent event) {
-        final PluginDO plugin = pluginMapper.selectById(event.getSelector().getPluginId());
-        List<DivideUpstream> existDivideUpstreams = SelectorUtil.buildDivideUpstream(event.getSelector(), plugin.getName());
-        if (CollectionUtils.isNotEmpty(existDivideUpstreams)) {
-            replace(event.getSelector().getId(), CommonUpstreamUtils.convertCommonUpstreamList(existDivideUpstreams));
-        }
-    }
-
-    /**
-     * listen {@link SelectorCreatedEvent} add data permission.
+     * listen {@link DiscoveryStreamUpdatedEvent} add data permission.
      *
      * @param event event
      */
-    @EventListener(SelectorUpdatedEvent.class)
```

Comment:
why remove this?

Reply:
this Event just Work Update Seletor#handler. Management of Upstream just change to
` eventPublisher.publishEvent(new DataChangedEvent(ConfigGroupEnum.DISCOVER_UPSTREAM, DataEventTypeEnum.UPDATE, Collections.singletonList(discoverySyncData)));`

Reply:
If We need to maintain the Selector#handler in database , I will restore the code

Reply:
I think removing it is the correct thing to do.

Reply:
For divide selectors, this can be removed, but for other non-divide selectors, this can't be removed.

Reply:
other selector may use selector update, it's necessary.